app/models/batch_connect/app.rb

```diff
@@ -102,6 +102,18 @@ def default_description
       ood_app.manifest.description
     end
 
+    def link
+      OodAppLink.new(
+        # FIXME: better to use default_title and "" description
```

Comment:
Would this fix take too long?

Reply:
No. It changes functionality though.

Reply:
I added a comment to https://github.com/OSC/ood-dashboard/issues/417 to address this.